tcl/pd_menus.tcl

```diff
@@ -575,6 +575,44 @@ proc ::pd_menus::create_preferences_menu {mymenu} {
         -command {::pd_menus::loadpreferences}
     $mymenu add command -label [_ "Forget All..."] \
         -command {::pd_menus::forgetpreferences}
+		
+	switch -- $::windowingsystem {
+       
+	   "win32" {
+	
+    $mymenu add command -label [_ "Reset .pd files to *this* Pd"] \
+        -command { 
+		
+		if {[file exists "$::sys_libdir/bin/wish86.exe"] == 1} { set wishN "wish86" } \
+		else { set wishN "wish85" }
```

Comment:
hmm. i think `[info nameofexecutable]` will give you the full name of the wish-executable.
no need to second-guess.